src/db/index.ts

```diff
@@ -148,6 +146,8 @@ export class LandRequest extends Model<LandRequest> implements ILandRequest {
       state,
       reason,
       prevState: prevStatus?.state,
+      stateDuration: prevStatus ? Date.now() - prevStatus.date.getTime() : null,
+      durationSinceQueued: queuedDate ? Date.now() - queuedDate.getTime() : null,
```

Comment:
durationSinceQueued should give cumulative queue time when LandRequestStatus `state` is `merging`?

Reply:
Yep, although `success` is slightly more accurate